packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.ts

```diff
@@ -577,6 +577,7 @@ declare function get_old_InterfaceDeclaration_IGeneratedSummaryStats():
 declare function use_current_InterfaceDeclaration_IGeneratedSummaryStats(
     use: TypeOnly<current.IGeneratedSummaryStats>);
 use_current_InterfaceDeclaration_IGeneratedSummaryStats(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
I think you are supposed to add entries in package.json, no?

Reply:
There is one entry I had to add, along with the tasks to actually generate the change in this file.